cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -997,13 +997,14 @@ public void deleteFlowQueues(HttpRequest request, HttpResponder responder,
 
   /**************************** Workflow/schedule APIs *****************************************************/
   @GET
-  @Path("/apps/{app-id}/workflows/{workflow-name}/current")
+  @Path("/apps/{app-id}/workflows/{workflow-name}/{run-id}/current")
```

Comment:
The purpose of the API is to get the information about what programs are currently in execution for the run specified by the run-id. This is different from the `status` which only returns whether the current program is `RUNNING` or `STOPPED`